.github/workflows/main.yml

```
@@ -50,8 +50,12 @@ jobs:
           fi
 
   is-release:
-    # release merge commits come from github-actions
-    if: startsWith(github.event.commits[0].author.name, 'github-actions')
+    # Filtering by `push` events ensures that we only release from the `main` branch, which is a
+    # requirement for our npm publishing environment.
+    # The commit author should always be 'github-actions' for releases created by the
+    # 'create-release-branch' workflow, so we filter by that as well to prevent accidentally
```

This doesn't sound right; instead of `create-release-branch` should this be `action-create-release-pr`?

---

Whoops! Good catch.

```suggestion
    # 'create-release-pr' workflow, so we filter by that as well to prevent accidentally
```

I meant to refer to the workflow name in this template, rather than the action, so I think this should be `create-release-pr`